tls/go-server/app/start.sh

```diff
@@ -0,0 +1,4 @@
+export CONF_PROVIDER_FILE_PATH="../conf/server.yml"
```

Comment:
avoid use shell script to start. try if build/makefile work or not. pls. refer to HOWTO.md